platform/broadcom/sonic-platform-modules-dell/debian/platform-modules-s5296f.install

```diff
@@ -4,6 +4,7 @@ s5296f/scripts/sensors usr/bin
 s5296f/scripts/pcisysfs.py usr/bin
 s5296f/cfg/s5296f-modules.conf etc/modules-load.d
 s5296f/systemd/platform-modules-s5296f.service etc/systemd/system
+s5296f/modules/sonic_platform-1.0-py3-none-any.whl usr/share/sonic/device/x86_64-dellemc_s5296f_c3538-r0
```

Comment:
whl should be a build artifacts, why do we install this in the folder. i think there is some problem here.

Reply:
Pls check the 'SONiC Platform API Package' comments in port guide.
The whl package will be installed later in the host and pmon with pip3 command.
Same procedure is followed in other vendors as well (accton,cel,...)

Reply:
@lguohan  
whl is build while making the platform-module package.
And while making the fsroot file system, the whl package will be copied to the device platform thru the install script of the the platform-module package